src/test/java/web_pages/TutByHomePage.java

```diff
@@ -17,6 +22,11 @@ public TutByHomePage(WebDriver driver){
 
     public void loginToSite(){
         driver.get("https://www.tut.by");
+        try {
+            takeScreenshot("src/test/resources/screen.png");
```

Comment:
according to taking screen - everything is ok, but, you should give a unique name to each screenshot, cause they would be rewritten each test, and you will not get the actual picture.